services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/RouterServiceImpl.java

```diff
@@ -261,18 +261,9 @@ public RouteTable getSubnetRouteTable(String projectId, String subnetId) throws
             routeTableMap = new HashMap<>();
         }
 
-        if (routeTableMap.size() == 0) {
-            // create a subnet route table
-            SubnetWebJson subnetWebJson = this.vpcRouterToSubnetService.getSubnet(projectId, subnetId);
-            String vpcId = subnetWebJson.getSubnet().getVpcId();
-            Router router = getOrCreateVpcRouter(projectId, vpcId);
-
-            String vpcDefaultRouteTableId = router.getVpcDefaultRouteTableId();
-            routeTable = this.routeTableDatabaseService.getByRouteTableId(vpcDefaultRouteTableId);
-            return routeTable;
-        } else if (routeTableMap.size() > 1) {
+        if (routeTableMap.size() > 1) {
```

Comment:
@kevin-zhonghao I think @DavidLiu506 has pretty much the same change in his PR. If most of the changes are already included in @DavidLiu506's PR, please just review @DavidLiu506's PR and ignore your current PR.